src/Tizen.Applications.Common/Tizen.Applications/DirectoryInfo.cs

```diff
@@ -180,5 +183,73 @@ public string ExpansionPackageResource
                 return _expansionPackageResourcePath;
             }
         }
+
+        /// <summary>
+        /// Gets the absolute path to the application's resource control directory, which is used to share the resources of the allowed resource packages that available.
```

Comment:
please change the final "that available" because it sounds bad and I don't understand it, maybe "available in the system" or just remove it, or sth like "available allowed resource package"
same at line 191, 222, 225

Reply:
done